fs/fs.go

```diff
@@ -410,10 +411,21 @@ func (i *RealFsInfo) GetFsInfoForPath(mountSet map[string]struct{}) ([]Fs, error
 			default:
 				var inodes, inodesFree uint64
 				if utils.FileExists(partition.mountpoint) {
+					if strings.HasPrefix(partition.fsType, "nfs") {
```

Comment:
Could you move NFS handling to dedicated case statement? I think it would increase code readability.

Reply:
@iwankgb done